packages/auth-server/composables/useRecoveryGuardian.ts

```diff
@@ -91,6 +91,25 @@ export const useRecoveryGuardian = () => {
     return tx;
   });
 
+  const { inProgress: getPendingRecoveryDataInProgress, error: getPendingRecoveryDataError, execute: getPendingRecoveryData, result: getPendingRecoveryDataResult } = useAsync(async (account: Address) => {
+    const client = getPublicClient({ chainId: defaultChain.id });
+    return await client.readContract({
+      address: contractsByChain[defaultChain.id].recovery,
+      abi: GuardianRecoveryModuleAbi,
+      functionName: "pendingRecoveryData",
+      args: [account],
+    });
+  });
+
+  const { inProgress: discardRecoveryInProgress, error: discardRecoveryError, execute: discardRecovery } = useAsync(async () => {
+    const client = getClient({ chainId: defaultChain.id });
+    return await client.writeContract({
```

Comment:
I believe we should add this method to sdk packages, inside packages/sdk/src/client/recovery/actions/recovery.ts actions file. WDYT?